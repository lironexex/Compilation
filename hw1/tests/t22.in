// comments 3 - (from the FAQ) - a single line comment as the last line of the input file
var x = 3
// this is legal