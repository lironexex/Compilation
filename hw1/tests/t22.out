1 COMMENT 1
2 VAR var
2 ID x
2 ASSIGN =
2 DEC_INT 3
3 COMMENT 1
